modules/reference/pages/audit/audit-subscription-matching.adoc

```diff
@@ -4,4 +4,56 @@
 The menu:Audit[Subscription Matching] section provides reports that match your currently installed clients to your existing product subscriptions.
 Subscription matching reports provide information about clients that do not have a subscription, and subscription start and end dates.
 
+
+
+[[subscription-matching-options]]
+[cols="1,1", options="header"]
+.Subscription Matching Options
+|===
+| Column          | Description
+| Part Number     | Identifier of the matched product
+| Description     | Description of the matched product
+| Policy          | The type of subscription matched to the product
+| Matched/Total   | The number of clients currently using the subscription, of the total available.
+If the subscription is fully matched, the quantity column value is highlighted.
+| Start Date      | Start date of the subscription
+| End Date        | End date of the subscription
+|===
+
+
+Table entries are highlighted if they are due to expire within three months.
+Table entries that have already expired are shown in grayscale.
+
+
+
+For messages relating to subscription matching, navigate to the [guimenu]``Messages`` tab.
+
+[[subscription-matching-status]]
+[cols="1,1,1", options="header"]
+.Subscription Matching Statuses
+|===
+| Status                  | Description | Action
+| Unsupported Part Number | The detected part number is unknown or unsupported. | Check the part number, and that the client has been matched to the correct subscription.
```

Comment:
The action for unsupported part numbers is to call support and open a Service Request. In this case we need to add the partnumber to the product.